docs/extend-graphql-schema.md

```diff
@@ -3,4 +3,568 @@ id: extend-graphql-schema
+recommend the reading of the Graphql documentation page
+[Thinking in Graphs](https://graphql.github.io/learn/thinking-in-graphs/).
+
+Let’s add the code to expose physical stores in our graph.
+
+### Define the schema
+
+Front-Commerce contains all the tooling to allow you to describe your schema
+using the expressive
+[GraphQL Schema Definition Language (SDL)](https://www.prisma.io/blog/graphql-sdl-schema-definition-language-6755bcb9ce51/).
+
+Create a `src/server/modules/physical-stores/schema.gql` file to contain the
+schema matching our use case. It could look like so:
+
+```gql
+type Query {
+  "Our physical stores"
+  physicalStores: [PhysicalStore]
+}
+
+"General information about a physical store"
+type PhysicalStore {
+  address: PostalAddress
+  openingHours: [OpeningHours]
+}
+
+"Postal address of a physical location"
+type PostalAddress {
+  street: String
+  city: String
+  postcode: String
+  country: String
+}
+
+"Opening hours for a given day"
+type OpeningHours {
+  day: DayOfWeek
+  openingTimes: [TimePeriod]
+}
+
+"Period of time within a day"
+type TimePeriod {
+  from: Time!
+  to: Time!
+}
+
+"24h time definition"
+type Time {
+  h: Int!
+  m: Int!
+}
+
+enum DayOfWeek {
+  MONDAY
+  TUESDAY
+  WEDNESDAY
+  THURSDAY
+  FRIDAY
+  SATURDAY
+  SUNDAY
+}
+```
+
+Then expose the schema using the `typeDefs` key of the module definition. Update
+the `src/server/modules/physical-stores/index.js` entrypoint:
+
+```diff
++import typeDefs from "./schema.gql";
++
+export default {
+-  namespace: "PhysicalStores"
++  namespace: "PhysicalStores",
++  typeDefs: typeDefs
+};
+```
+
+> Webpack is in fact taking care of converting SDL into a Javascript object
+> using an appropriate loader.
+
+**Congratulations again!** You should now be able to see these new types and
+query them in GraphQL.
+
+Try to execute the query below in Graph*i*QL:
+
+```
+{
+  physicalStores {
+    address {
+      street
+      city
+      country
+      postcode
+    }
+    coordinates {
+      latitude
+      longitude
+    }
+    openingHours {
+      day
+      openingTimes {
+        from {
+          h
+          m
+        }
+        to {
+          h
+          m
+        }
+      }
+    }
+  }
+}
+```
+
+Our GraphQL module does not yet have any code for sending content, so you must
+see the following response:
+
+```json
+{
+  "data": {
+    "physicalStores": null
+  }
+}
+```
+
+Let’s now add some executable logic to our module.
+
+### Implement resolvers
+
+The GraphQL middleware relies on resolver functions to determine the data to
+return for a given field. This is where most of the « real work » is done, for
+instance by fetching remote datasources or transforming data.
+
+Resolvers are exposed using the `resolvers` key of the module definition. It
+should be a « resolver map »: an object where each key is a GraphQL type name,
+and values are mapping between field names and resolver function. Resolver
```

Comment:
I don't understand your definition of `resolver map`.

Are `« those quotes »` valid in english?

Reply:
I don't think they are valid :slightly_frowning_face: